integration_tests/test_benchmark.py

```diff
@@ -68,17 +71,17 @@ def _run_critcmp():
     print('ERRORS')
     print(p.stderr.decode('utf-8'))
 
-def _checkout_master_branch():
+def _git_checkout_upstream_branch():
     subprocess.run(
-        "git fetch {} {}".format(REMOTE, BRANCH),
+        "git fetch {} {}".format(GIT_REMOTE_NAME, BASE_BRANCH),
```

Comment:
nit: from what it looks like, fetching works with links too, so GIT_REMOTE_NAME could be 'BUILDKITE_REPO' here too.

Reply:
Good idea. Will do.